BetterThanRottenTomatoAPI/ReviewManagement/ReviewManagement.Infrastructure/Repositories/ReviewRepository.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Dapper;
+using Microsoft.Extensions.Configuration;
+using ReviewManagement.Infrastructure.Repositories.Entities;
+using ReviewManagement.Infrastructure.Repositories.Interfaces;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ReviewManagement.Infrastructure.Repositories
+{
+    public class ReviewRepository : IReviewRepository
+    {
+        private readonly string _connectionString;
+
+        public ReviewRepository(IConfiguration config)
+        {
+            _connectionString = config.GetSection("ConnectionString:TomatoDb").Value;
```

Comment:
Reason this did not work is because you did "ConnectionString" rather than "ConnectionString**s**" to match with what you input. But what you've added is duplicate since we already have a ConnectionStrings section is appsetting.development.json. So what you can do is change it to config.GetSection("ConnectionStrings:BetterThanRottenTomato").Value or config.ConnectionStrings("BetterThanRottenTomato").